File: Entities/DischargeSummary.cs
namespace PrivateHospitalSystem.Entities
{
    public class DischargeSummary
    {
        public Guid Id { get; set; }

        public Guid AdmissionId { get; set; }
        public Admission? Admission { get; set; }

        public string Diagnosis { get; set; } = string.Empty;
        public string TreatmentSummary { get; set; } = string.Empty;
        public string? MedicationOnDischarge { get; set; }
        public DateTime? FollowUpDate { get; set; }

        public Guid IssuedByDoctorId { get; set; }
        public Doctor? IssuedByDoctor { get; set; }

        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}

File: frontend/src/types/dischargeSummary.ts
export interface DischargeSummary {
  id: string
  admissionId: string
  patientName: string
  diagnosis: string
  treatmentSummary: string
  medicationOnDischarge?: string
  followUpDate?: string
  issuedByDoctorName: string
  issuedAt: string
}

export interface CreateDischargeSummaryDto {
  admissionId: string
  diagnosis: string
  treatmentSummary: string
  medicationOnDischarge?: string
  followUpDate?: string
  issuedByDoctorId: string
}
